org.hlwd.sonofman.yml

```diff
@@ -0,0 +1,29 @@
+app-id: org.hlwd.sonofman
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: som
+finish-args:
+  - --filesystem=home
```

Comment:
It should use $XDG_DATA_HOME to save configuration file. The applications have write access there by default.